IRremoteESP8266.cpp

```diff
@@ -201,7 +201,6 @@ void IRsend::sendLG (unsigned long data, int nbits) {
   // Header
   mark(LG_HDR_MARK);
   space(LG_HDR_SPACE);
-  mark(LG_BIT_MARK);
```

Comment:
I agree it looks odd/different. However, the existing code's loop is based on "space then mark", rather than the typical "mark then space". sendData() uses the later.
It turns out that to convert it to a "mark then space" loop, all you need to do is shift the preceding bit mark to the end.

e.g.
Lets assume a 2 bit data of 0x01 we are sending. The loop/sendData() sections denoted by '[]'s.

Old:
M(LG_HDR_MARK), S(LG_HDR_SPACE), M(LG_BIT_MARK), [ S(LG_ZERO_SPACE), M(LG_BIT_MARK), S(LG_ONE_SPACE), M(LG_BIT_MARK) ], S(0)

New:
M(LG_HDR_MARK), S(LG_HDR_SPACE), [ M(LG_BIT_MARK), S(LG_ZERO_SPACE), M(LG_BIT_MARK), S(LG_ONE_SPACE) ], M(LG_BIT_MARK), S(0)